docs/UserGuide.md

```diff
@@ -17,28 +17,48 @@ MindMap is a **desktop app for managing contacts, optimized for use via a  Line
 
 1. Ensure you have Java `17` or above installed in your Computer.
```

Comment:
May be useful to link this: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html